src/components/Modal.js

```diff
@@ -0,0 +1,13 @@
+export default {
+	methods: {
+		toggleModal() {
+			// Quick and clean way to emit to grandparent
+			// that modal should be opened/closed
+			let vm = this;
+			while (vm) {
```

Comment:
Is this duplicated on purpose? 🐟

Reply:
Yes, because I need it here too :P

I moved this to a mixin now!